### DISCUSSION TITLE
Automated backups don't include media files?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#7292] [Q&A]
Is this by design? I've got automated backups enabled and run daily. Only a DB dump is done and backed up. 

Should the media files also be added in the backups or do I need to take care of this in a separate backup script?

Running 0.15.1 on baremetal/installer script.

**Comment by SchrodingersGat** [comment]
The automated backup *should* be doing the media files too:

https://github.com/inventree/InvenTree/blob/bb93c915eeb06e353aec07385733467a3ad5683d/src/backend/InvenTree/InvenTree/tasks.py#L637

Maybe you have check your background worker logs to see if there are any clues?

**Comment by StormTrooper** [comment]
Are there specific logs for the worker process? Where do I find them?

Even stranger today there are no backups.
Worker says it was successful:

```
Stopped:
May 22, 2024, 11:32 a.m.
```

But then no backups are visible for today. Only thing I see in the syslog is:


```
May 22 11:32:56 inventree-server inventree-worker-1.service[41509]: /opt/inventree/env/lib/python3.9/site-packages/django/db/models/fields/__init__.py:1535: RuntimeWarning: DateTimeField PartPricing.updated received a naive datetime (2024-04-22 00:00:00) while time zone support is active.
May 22 11:32:56 inventree-server inventree-worker-1.service[41509]:   warnings.warn(
```

Update: I'm being a idiot :) There is a ```inventree-server-2024-05-21-113304.tar.gz``` file for yesterday which contains the media files. 

However nothing for today though - no media or DB backup

**Comment by StormTrooper** [comment]
Closing this as it's working as expected. Backups ran today.
